services/mgmt/arcdata/README.md

```diff
@@ -6,12 +6,12 @@ https://github.com/Azure/azure-rest-api-specs/blob/main/specification/azurearcda
 
 To get started with these generated service crates, see the [examples](https://github.com/Azure/azure-sdk-for-rust/blob/main/services/README.md#examples).
 
-The default tag is `package-preview-2023-03`.
```

Comment:
Scanning through a selection of the changes, I was a little surprised to see some of the latest versions switch to an older version.  However, it seems that this is just due (in this case at least) to the newer API being removed, e.g. for `ArcData`:

- https://github.com/Azure/azure-rest-api-specs/pull/23821/files

So this appears to be fine.